packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -585,6 +598,116 @@ class SingleActivator with Diagnosticable implements ShortcutActivator {
   }
 }
 
+/// A shortcut combination that is triggered by a key event of a character.
+///
+/// Keys often produce different characters when combined with modifiers. For
+/// example, it might be helpful for the user to bring up a help menu by
+/// pressing the question mark ('?'). However, there is no logical key that
+/// directly represents a question mark. Althouh 'Shift+Slash' produces a '?'
+/// character on a US keyboard, its logical key is still considered a Slash key,
+/// and hard-coding 'Shift+Slash' in this situation is unfriendly to other
+/// keyboard layouts.
+///
+/// The [CharacterActivator] is useful when the shortcut intent should be
+/// activated upon an event that produces a character undefined by a logical
+/// key. For example, `CharacterActivator('?')` is triggered when a key
+/// combination results in a question mark, which is 'Shift+Slash' on a US
+/// keyboard, but 'Shift+Comma' on a French keyboard.
+///
+/// The [CharacterActivator] is less reliable than [SingleActivator] since
+/// it depends on the platform to report event characters. Prefer
+/// [SingleActivator] if the desired key combination can be described with
+/// logical keys.
+///
+/// {@tool dartpad --template=stateful_widget_scaffold_center}
+/// In the following example, when a key combination results in a question mark,
+/// the counter is increased:
+///
+/// ```dart preamble
+/// class HelpMenuIntent extends Intent {
+///   const HelpMenuIntent();
+/// }
+/// ```
+///
+/// ```dart
+/// int count = 0;
```

Comment:
This example would be better if it did something other than increment a counter, especially since the intent is called `HelpMenuIntent`. A better example would be to have it pop up a SnackBar with some fake help or something. Since you're using a template with a scaffold, that should be pretty easy:

```dart
ScaffoldMessenger.of(context).showSnackBar(
  SnackBar(content: Text('You need help!")),
);
```

Reply:
I failed to come up with an easy way like this. Thanks!